website/docs/plugins/ax_sweeper.md

```diff
@@ -42,22 +42,22 @@ The output of a run looks like:
 
 ```
 [HYDRA] AxSweeper is optimizing the following parameters:
-banana.x: range=[-5, 5], type = int
-banana.y: range=[-5.0, 10.1], type = float
-ax.modelbridge.dispatch_utils: Using Bayesian Optimization generation strategy: GenerationStrategy(name='Sobol+GPEI', steps=[Sobol for 5 arms, GPEI for subsequent arms], generated 0 arm(s) so far). Iterations after 5 will take longer to generate due to model-fitting.
-AxSweeper is launching 5 jobs
+banana.x: range=[-5, 5]
+banana.y: range=[-5.0, 10.1]
+ax.modelbridge.dispatch_utils: Using Bayesian Optimization generation strategy: GenerationStrategy(name='Sobol+GPEI', steps=[Sobol for 5 trials, GPEI for subsequent trials]). Iterations after 5 will take longer to generate due to  model-fitting.
+[HYDRA] AxSweeper is launching 5 jobs
 [HYDRA] Launching 5 jobs locally
-[HYDRA]   #0 : banana.x=4 banana.y=-1.484
-[__main__][INFO] - Banana_Function(x=4, y=-1.484)=30581.473
-[HYDRA]   #1 : banana.x=3 banana.y=-3.653
-[__main__][INFO] - Banana_Function(x=3, y=-3.653)=16014.261
-[HYDRA]   #2 : banana.x=0 banana.y=9.409
-[__main__][INFO] - Banana_Function(x=0, y=9.409)=8855.340
-[HYDRA]   #3 : banana.x=-4 banana.y=2.059
-[__main__][INFO] - Banana_Function(x=-4, y=2.059)=19459.063
-[HYDRA]   #4 : banana.x=-3 banana.y=-1.338
-[__main__][INFO] - Banana_Function(x=-3, y=-1.338)=10704.497
-[HYDRA] New best value: 8855.340, best parameters: {'banana.x': 0, 'banana.y': 9.409}
+[HYDRA]        #0 : banana.x=2 banana.y=-0.9878678928129379
+[__main__][INFO] - Banana_Function(x=2, y=-0.9878678928129379)=2488.8826116154178
+[HYDRA]        #1 : banana.x=-1 banana.y=7.700736676901579
+[__main__][INFO] - Banana_Function(x=-1, y=7.700736676901579)=4493.987201317402
+[HYDRA]        #2 : banana.x=-1 banana.y=-3.901284400280565
+[__main__][INFO] - Banana_Function(x=-1, y=-3.901284400280565)=2406.2588772433614
+[HYDRA]        #3 : banana.x=-1 banana.y=0.20855206744745391
+[__main__][INFO] - Banana_Function(x=-1, y=0.20855206744745391)=66.63898299416996
+[HYDRA]        #4 : banana.x=4 banana.y=-4.557481761090457
+[__main__][INFO] - Banana_Function(x=4, y=-4.557481761090457)=42270.00563575668
+[HYDRA] New best value: 66.63898299416996, best parameters: {'banana.x': -1, 'banana.y': 0.20855206744745391}
```

Comment:
curious why did this output change so much?

Reply:
additionally, can we truncate the values to 3 places of decimal?

Reply:
> curious why did this output change so much?

Trials is the correct term here rather than arms. Other than that, it just looks like some whitespace and an increase in sig figs, and I'm not sure why that changed. As for the actual numerical values changing, I suppose that could be due to randomness in each run, as well as the change implemented here, namely, setting SEM to 0.

> additionally, can we truncate the values to 3 places of decimal?

I could be wrong but this sounds like something that needs to happen in the Ax codebase, which I think should be decoupled from this PR and done later. I can truncate manually if you like in the mean time.

Reply:
Regarding truncating, I meant manually truncating in the docs (not the code etc). Sorry I should have clarified it earlier.